### DISCUSSION TITLE
Enable quick compaction-checker

### DISCUSSION THREAD

**Discussion by xiaobiaozhao** [apache/kvrocks#1282] [General]
In some case，we need save a lot of data with short ttl, eg ttl = 3600s. 
In current kvrocks, it takes at least two days for compation to take place, which can cause disk space not to be released quickly. I think kvrocks needs to come up with a configuration to figure out how long compation will take.

**Comment by git-hulk** [comment]
@xiaobiaozhao You should describe your motive, I think no one can get your point with this title.
